apis/server/container_bridge.go

```diff
@@ -216,28 +218,48 @@ func (s *Server) attachContainer(ctx context.Context, rw http.ResponseWriter, re
 }
 
 func (s *Server) getContainers(ctx context.Context, rw http.ResponseWriter, req *http.Request) error {
+	option := &mgr.ContainerListOption{
+		All: httputils.BoolValue(req, "all"),
```

Comment:
If we support a flag in daemon side, I think it is changing API, then we should consider updating swagger.yml in parameter part like:
```
        - name: "all"
          in: "query"
          description: "Return all containers. By default, only running containers are shown"
          type: "boolean"
          default: false
```
In API endpoint `GET /containers/json` in swagger.yml.